ast/builtins.go

```diff
@@ -2289,6 +2290,17 @@ var CryptoX509ParseCertificateRequest = &Builtin{
 	),
 }
 
+var CryptoX509ParseKeyPair = &Builtin{
+	Name:        "crypto.x509.parse_x509_keypair",
```

Comment:
Let's change `crypto.x509.parse_x509_keypair` to something like `crypto.x509.parse_keypair` to stay consistent with the other builtin names in that category.